pr_check.sh

```diff
@@ -11,6 +11,6 @@ make build
 # verify if schema update causes issues
 make update-schema validate-schema
 
-# We must use the same version as the terraform provider here 
-grep -q 'github.com/keybase/go-crypto v0.0.0-20161004153544-93f5b35093ba' go.mod || (echo "go-crypto version mismatch"; exit 1)
+# We must use the same version as the terraform provider here.
```

Comment:
**To the reviewer:**

I am not sure if this is useful? Thoughts?

Reply:
ok for me